x-pack/plugins/lists/server/services/exception_lists/exception_list_client_types.ts

```diff
@@ -112,7 +114,7 @@ export interface UpdateExceptionListItemOptions {
   description: DescriptionOrUndefined;
   meta: MetaOrUndefined;
   tags: TagsOrUndefined;
-  type: ExceptionListTypeOrUndefined;
+  type: ExceptionListItemTypeOrUndefined;
```

Comment:
Before both the list and item types were just t.string so it didn't scream at us, but now that the list type and item type differ, had to update.